app/scenes/right_menu_drawer/right_menu_drawer.js

```diff
@@ -163,7 +186,7 @@ export default class RightMenuDrawer extends React.Component {
                         defaultMessage='Help'
                     />
                 </RightMenuDrawerItem>
-                <RightMenuDrawerItem>
+                <RightMenuDrawerItem onPress={this.openErrorEmail}>
```

Comment:
This option should be enabled only if there are some errors in the store, and after that the error store should be cleared to avoid sending the same errors all the time